File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Exercise/07_Shopping.js
function shopping(input) {
    let videoCardPrice = 250;

    let budget = Number(input[0]);
    let videoCardsCount = Number(input[1]);
    let processorsCount = Number(input[2]);
    let ramCount = Number(input[3]);

    let videoCardsSum = videoCardsCount * videoCardPrice;
    let processorPrice = videoCardsSum * 0.35;
    let ramPrice = videoCardsSum * 0.10;

    let processorsSum = processorsCount * processorPrice;
    let ramSum = ramCount * ramPrice;

    let totalSum = videoCardsSum + processorsSum + ramSum;

    if (videoCardsCount > processorsCount) {
        totalSum *= 0.85;
    }

    if (budget >= totalSum) {
        console.log(`You have ${(budget - totalSum).toFixed(2)} leva left!`);
    } else {
        console.log(`Not enough money! You need ${(totalSum - budget).toFixed(2)} leva more!`);
    }
}

// shopping(["900", "2", "1", "3"]);

File: Python_Courses/Python_Basics/Python_PB_2025_01/02_Conditional_Statements_Exercise_OFF/07_Shopping.py
budget = float(input())
video_cards_count = int(input())
processors_count = int(input())
ram_count = int(input())

video_cards_price = video_cards_count * 250

processor_price = video_cards_price * 0.35
ram_price = video_cards_price * 0.10

total_price = video_cards_price + processors_count * processor_price + ram_count * ram_price

# Ако броя на видеокартите е по-голям от този на процесорите получава 15% отстъпка от крайната сметка
if video_cards_count > processors_count:
    total_price *= 0.85

if budget > total_price:
    money_left = budget - total_price
    print(f'You have {money_left :.2f} leva left!')
else:
    money_needed = total_price - budget
    print(f'Not enough money! You need {money_needed :.2f} leva more!')
